src/main/java/arrow/RaiseAddImport.java

```diff
@@ -0,0 +1,55 @@
+package arrow;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import lombok.EqualsAndHashCode;
+import lombok.Value;
+import org.openrewrite.ExecutionContext;
+import org.openrewrite.Recipe;
+import org.openrewrite.TreeVisitor;
+import org.openrewrite.java.tree.J;
+import org.openrewrite.kotlin.KotlinIsoVisitor;
+
+@Value
+@EqualsAndHashCode(callSuper = true)
+public class RaiseAddImport extends Recipe {
+    @JsonCreator
+    public RaiseAddImport() {
+    }
+
+    @Override
+    public String getDisplayName() {
+        return "Raise Rewrite";
+    }
+
+    @Override
+    public String getDescription() {
+        return "Rewrites all imports, and builders from arrow.core.computations.* and arrow.core.continuations.* to arrow.core.raise.*.";
+    }
+
+    @Override
+    protected TreeVisitor<?, ExecutionContext> getVisitor() {
+        return new RaiseImportVisitor();
+    }
+
+    public class RaiseImportVisitor extends KotlinIsoVisitor<ExecutionContext> {
+
```

Comment:
```suggestion
        MethodMatcher effectScopeMatcher = new MethodMatcher("arrow.core.continuations.EffectScope ensure(..)");
        MethodMatcher eagerEffectScopeMatcher = new MethodMatcher("arrow.core.continuations.EagerEffectScope ensure(..)");
```

Reply:
Rather than `isEnsureInvocation` you may use `MethodMatcher` which contains APIs to check the `tree` or `MethodType`.